acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/multikeysigner/MultiKeyAcceptanceTestBase.java

```diff
@@ -52,14 +52,10 @@ void setup(final Path tomlDirectory) {
   }
 
   void createAzureTomlFileAt(
-      final String tomlFilename,
-      final String clientId,
-      final String clientSecret,
-      final Path tomlDirectory) {
+      final Path absTomlPath, final String clientId, final String clientSecret) {
```

Comment:
nit: is the distinction that this is the absolute path important for this function?

Reply:
true - changed back to tomlPath